end-to-end-tests/src/helpers/ctx.rs

```diff
@@ -5,7 +5,7 @@ use omicron_sled_agent::rack_setup::config::SetupServiceConfig;
 use omicron_test_utils::dev::poll::{wait_for_condition, CondCheckError};
 use oxide_client::types::{Name, ProjectCreate};
 use oxide_client::CustomDnsResolver;
-use oxide_client::{Client, ClientProjectsExt, ClientVpcsExt};
+use oxide_client::{Client, ClientProjectsExt, ClientVpcsSubnetsExt};
```

Comment:
this isn’t supposed to be here, is it?

Reply:
oh I see, this actually does come from the tags

Reply:
yes sir